libs/coin-framework/src/bridge/jsHelpers.ts

```diff
@@ -221,6 +221,8 @@ export const makeSync =
             syncConfig
           );
 
+          log("bridge", "getAccountShape", shape);
```

Comment:
Logging that without conditions might be problematic as log file size might be exploding on big accounts. I'm not sure this one will help on our investigation 🤔